modules/product/src/main/java/com/opengamma/strata/product/fx/FxVanillaOption.java

```diff
@@ -103,31 +88,13 @@
    */
   @PropertyDefinition(validate = "notNull")
```

Comment:
At expiry, the long party will have the option to enter in this transaction; the short party will, at the option of the long party, potentially enter into the inverse transaction.